src/diff/component.js

```diff
@@ -163,7 +168,7 @@ export function renderComponent(
 
 	let nextDomSibling;
 
-	if (isNew) {
+	if (internal._children == null) {
```

Comment:
This change is necessary for suspense hydration. Since the suspending node will have a valid internal and component instance after the first render (the one that suspends), `isNew` will be `false` on subsequent renders. However, particularly for suspense hydration we want the children to be mounted/hydrated even though the component isn't new. So using this condition to detect that scenario which works for both `isNew == false` and suspense hydration.

Reply:
I'm most excited about changing that component instance thing in `Suspense`. Should make things a lot easier 👍